safe/definitions/hazard_classifications.py

```diff
@@ -638,16 +638,14 @@
             'name': tr('Use caution'),
             'affected': False,
             'description': tr(
-                'No water encountered above ground height but there are may '
-                'be floods in surrounding areas and you should use caution '
-                'when moving around in this area.'),
+                'An unknown level of flooding - use caution - '),
             'fatality_rate': None,
             # displacement rate estimated from DMI analysis of historical
             # flood data and IDP numbers
             'displacement_rate': 0.0,
             'numeric_default_min': 0,
             'numeric_default_max': 0.1,
-            'string_defaults': ['caution'],
+            'string_defaults': ['caution, unknown'],
```

Comment:
If you want to register two strings, you should put `'` in between the string, before adding comma. So, it should be `['caution', 'unknown']`